string address
float64 rate_set_point
float64 rate_actual
bool success

